apps/full-site-editing/full-site-editing-plugin/block-inserter-modifications/contextual-tips/utils.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * Detect if the editor is already iFramed.
+ *
+ * @returns {boolean} `True` is the editor is iFramed. Otherwise, `False`.
+ */
+export const inIframe = () => {
+	try {
+		return window.self !== window.top;
+	} catch ( e ) {
+		return true;
+	}
+};
+
+export const isSimpleSite = !! (
+	window &&
+	window._currentSiteType &&
+	window._currentSiteType === 'simple'
+);
```

Comment:
These seem like handy utils in general, would there be good place to share these within `full-site-editing` vs have them specific to `contextual-tips`?

Reply:
Let's explore that once there's a need for it in another module?

Reply:
yes... my concern is FSE is like a plugins container, so in theory, are they independent plugins?
I mean is it possible that, at some moment, we could move some component outside of FSE? For instance, moving `block-inserter-modifications` to `wpcom-block-editor`.

Reply:
The movement would be between Gutenberg/Jetpack/FSE plugin and at that point, you're in for a pretty big refactor anyway. Util or two won't matter much at that point :-)

As for `wpcom-block-editor`; we should roll it into FSE plugin as it's basically reduntant.